### PR TITLE
Sections recent works and about

### PR DESCRIPTION
**Pull request by AKeeganDev**
ADD RECENT WORKS section (utilizing grids)
ADD ABOUT ME section
Various stylings and font imports

This is intended to be viewed on a 375px resolution window. Code is not yet optimized for a full-size browser window at this time!

### CODE REVIEW

**Review by prolajumokeoni — CHANGES_REQUESTED** [review]
Hi @AKeeganDev , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there 🥇 

## Highlights
- Github flow was used correctly ✔️ 
- No linter errors ✔️ 


## Required Changes ♻️
_Check the comments under the review._

- [ ] Kindly add the [About myself](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md) section; Languages, Frameworks, and Skills please check the requirements 👍🏿 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Kindly capitalize the first word and use imperative mood for commit messages, kindly check this article on how to write [good commit messages](https://www.freecodecamp.org/news/how-to-write-better-git-commit-messages/) 👍🏿
- Kindly use [Grammarly](https://app.grammarly.com/) to run check for grammatical, punctuation, and spelling errors, this helps your pull request message be more professional 🤗 


almost-there
Cheers and Happy coding!👏👏👏
-----------------------------------------------
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -32,5 +33,212 @@ <h3 class="who-am-i">I am a software Developer</h3>
...
+                  Ruby
+                </div>
+              </li>
+            </ul>
+          </div>
+        </div>
+        <button class="see-project card-button"><span>See Project</span></button>
+      </div>
+    </div>
+  </main>
```
[the remark this answers is not in the document]
Comment (prolajumokeoni) [reply index.html:242]: Hi @AKeeganDev, I am not referring to the cards, I am referring to the different sections of your project. You currently have a main tag, so for sections like about myself it should have a section tag wrapping all elements, also for works it should have its own section tags

[the remark this answers is not in the document]
Comment (prolajumokeoni) [reply index.html:242]: Kindly tag me on this thread if you need more clarifications @AKeeganDev

**Comment by AKeeganDev** [comment]
@prolajumokeoni One last question. You mentioned that I relook at the About Me Section requirements again. I looked and they state that my Languages, Skills, etc. need to be in a <ul> using <li>s. I have indeed done that. I am not sure what you're asking me to change. If you could clarify that would be great. Thank you again for your review and for replying so quickly to my messages!

File: css/styles.css [hunk 2]
```diff
@@ -90,6 +90,15 @@ body {
   max-width: 200px;
 }
 
+.tags li {
+  font-family: 'Inter', sans-serif;
+  list-style: none;
+  border: 1px solid #091e42;
```
[the remark this answers is not in the document]
Comment (AKeeganDev) [reply css/styles.css:96]: @prolajumokeoni 

Sorry if you got a 2nd notification for this. I moved my last message to this thread. I have another question, please. 

You mentioned that I relook at the About Me Section requirements again. I looked and they state that my Languages, Skills, etc. need to be in a < ul > using < li >s. I have indeed done that. I am not sure what you're asking me to change. If you could clarify that would be great. Thank you again for your review and for replying so quickly to my messages!

Cheers!

File: index.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (AKeeganDev) [reply index.html:242]: @prolajumokeoni 

sorry if you got multiple notifications for this. I reposted the message to this thread. I am still getting used to Github!

One more question, please. You mentioned that I relook at the About Me Section requirements again. I looked and they state that my Languages, Skills, etc. need to be in a < ul > using < li >s. I have indeed done that. I am not sure what you're asking me to change. If you could clarify that would be great. Thank you again for your review and for replying so quickly to my messages!

**Comment by prolajumokeoni** [comment]
Hi @AKeeganDev when I reviewed your project you did not have the [About myself](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md) section containing Languages, Frameworks, and Skills please check the requirements 👍🏿

**Comment by prolajumokeoni** [comment]
Sure, we can  👍🏿

**Comment by AKeeganDev** [comment]
> Sure, we can 👍🏿

@prolajumokeoni  I simply forgot to push it last night :( my local branch was ahead by 1 commit. How embarrassing! I just pushed it in so I will ask for a review of it on my next pull request and save you some time for now. Thank you for being so attentive to my questions. I will make the changes you requested and also make sure the About Me section is included in the next pull request.

Thank you for all of your help and attentiveness. Also thank you for being willing to speak to me. I think for now I will finish my changes and move forward to my next pull request.

File: css/styles.css [hunk 3]
```diff
@@ -47,7 +47,7 @@ body {
   background-position: 100% 115%;
   color: var(--white-theme);
   padding: 120px 5%;
-  min-width: 375px;
+  max-width: 375px;
   flex: 0 0 auto;
```
[the remark this answers is not in the document]
Comment (AKeeganDev) [reply css/styles.css:51]: I will attempt to fix this before I merge. Thank you for the insight and thank you for the review. I am happy to have this completed. Cheers!

File: css/styles.css [hunk 4]
```diff
@@ -105,3 +114,227 @@ body {
...
   text-decoration: none;
 }
+
+.recent-works {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  padding: 120px 24px;
+  max-width: 375px;
+}
```
[the remark this answers is not in the document]
Comment (AKeeganDev) [reply css/styles.css:124]: @Wes-Isaac 

Okay!! Thank you!

**Pull request merged**